install/kubernetes/helm/subcharts/operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
That's another thing I hope we can avoid in future - if we remove the extra labels we also don't need this template.